core/domain/email_manager.py

```diff
@@ -163,6 +163,28 @@ def log_new_error(*args, **kwargs):
 
 
 def _require_sender_id_is_valid(intent, sender_id):
+    """Only allow emails as specified in the config.
+
+    Many emails are only allowed to be sent by certain user
+    or type of user, e.g. 'admin' or an admin/moderator. This
+    function will raise an exception if the email is not allowed.
+
+    Args:
+        intent: str. The intent string, i.e. the purpose of the email.
+                    Valid intent strings are defined in feconf.py
+        sender_id: str. The ID of the user sending the email, for example to
+                    check if the user is a moderator/admin or the system user.
+
+    Returns:
+        None
+
+    Raises:
+        Exception: Invalid email intent string.
```

Comment:
Our current format is:

```
    Raises:
        Exception: The email intent is invalid.
        Exception: The sender_id is not appropriate for the given intent.
```

(I.e., just the type of exception, and the circumstances that cause it to occur.)